docker-compose.yml

```diff
@@ -1,20 +1,22 @@
 version: "3"
 services:
-  editor:
-    build:
-      context: .
-      args:
-        USE_FIXTURES: 'false'
-        TRELLIS_BASE_URL: http://localhost:8080
-    environment:
-      INDEX_URL: http://search:9200
-    ports:
-      - 8000:8000
-    depends_on:
-      - platform
-      - pipeline
+  # editor:
+  #   build:
+  #     context: .
+  #     args:
+  #       USE_FIXTURES: 'false'
+  #       TRELLIS_BASE_URL: http://localhost:8080
+  #   environment:
+  #     INDEX_URL: http://search:9200
+  #   ports:
+  #     - 8000:8000
+  #   depends_on:
+  #     - platform
+  #     - pipeline
   pipeline:
-    image: ld4p/sinopia_indexing_pipeline:latest
+    # image: ld4p/sinopia_indexing_pipeline:latest
```

Comment:
Is there a reason for the switch from a published image for the indexing pipeline? Using a relative path assumes that the indexing pipeline is available up a directory and that it is up to date. I think we want to keep using published Docker images.

Reply:
This is was errant commit. Will fix.